nixos/modules/services/desktops/espanso.nix

```
@@ -15,16 +15,36 @@ in {
     };
   };
 
-  config = mkIf cfg.enable {
-    systemd.user.services.espanso = {
-      description = "Espanso daemon";
-      serviceConfig = {
-        ExecStart = "${lib.getExe cfg.package} daemon";
-        Restart = "on-failure";
-      };
-      wantedBy = [ "default.target" ];
-    };
+  config =
+    let
+      wayland = cfg.package == pkgs.espanso-wayland;
```

It might be better to do something like
```suggestion
      wayland = builtins.elem "wayland" cfg.package.buildFeatures;
```
Otherwise this will probably not work if one pulls an `espanso-wayland` package from another nixpkgs branch (e.g., if one runs nixos-2405 but pulls espanso-wayland from nixpkgs-unstable).